CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## Added
 
 * Support for nodeSelectors
-* Support for Ingress (UI only)
+* Support for UI and SQL Ingress 
```

Comment:
This doesn't seem right. We didn't add this in v2.3.0. I think we should add a new entry in the Unreleased section rather than changing this one.

Reply:
Moved to correct location